Examples/SubscriberExample/Sources/Utils/LocationAnimator/DefaultLocationAnimator.swift

```diff
@@ -0,0 +1,250 @@
+import Combine
+import AblyAssetTrackingCore
+import QuartzCore
+
+class DefaultLocationAnimator: NSObject, LocationAnimator {
+    
+    var infrequentlyUpdatingPositionInterval: TimeInterval = 5.0
+    
+    // Default values
+    private let intentionalAnimationDelay: TimeInterval = 2.0
+    private let defaultDisplayLinkDuration: CFTimeInterval = 1.0/60.0
+    
+    // Dispatch queue for synchronized variable access
+    private let globalBackgroundSyncronizeDataQueue = DispatchQueue(label: "com.ably.tracking.SubscriberExample.locationAnimator.globalBackgroundSyncronizeSharedData")
+    
+    // Dispatch queue for animation calculations
+    private let processAnimationQueue = DispatchQueue(label: "com.ably.tracking.SubscriberExample.locationAnimator.processAnimationQueue")
+
+    private var displayLinkStartTime: CFAbsoluteTime = .zero
+    private var animationRequestSubject = PassthroughSubject<AnimationRequest, Never>()
+    private var subscriptions = Set<AnyCancellable>()
+    private var animationStartTime: CFAbsoluteTime = .zero
+    private var animationStepStartTime: CFAbsoluteTime = .zero
+    private var currentAnimationStepProgress: Double = 1.0
+    private var currentAnimationStepIndex: Int = -1
```

Comment:
This property is no longer used anymore, probably a leftover after some latest commits.

Reply:
@JakubJankowski Are you able to make a commit removing this if it's not used anymore (and if you think it's not necessary please?

Reply:
Yeah, will do!

Reply:
I see you have fixed this here https://github.com/ably/ably-asset-tracking-swift/pull/289/commits/6dc6961b3c8cfc8491d20cb721715432befe3584, so will resolve this conversation